build/SignToolData.json

```diff
@@ -0,0 +1,246 @@
+{
+  "sign": [
+    {
+      "certificate": "MicrosoftSHA2",
+      "values": [
+        "bin/Microsoft.AspNetCore.Blazor.Analyzers/netstandard1.3/Microsoft.AspNetCore.Blazor.Analyzers.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Browser.JS/netcoreapp2.1/Microsoft.AspNetCore.Blazor.Browser.JS.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Browser/netstandard2.0/Microsoft.AspNetCore.Blazor.Browser.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Server/netstandard2.0/Microsoft.AspNetCore.Blazor.Server.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Build/netcoreapp2.1/Microsoft.AspNetCore.Blazor.Build.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Cli/netcoreapp2.1/dotnet-blazor.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Razor.Extensions/net461/Microsoft.AspNetCore.Blazor.Razor.Extensions.dll",
+        "bin/Microsoft.AspNetCore.Blazor.Razor.Extensions/netstandard2.0/Microsoft.AspNetCore.Blazor.Razor.Extensions.dll",
+        "bin/Microsoft.AspNetCore.Blazor/netstandard2.0/Microsoft.AspNetCore.Blazor.dll",
+        "bin/Microsoft.JSInterop/netstandard2.0/Microsoft.JSInterop.dll",
+        "bin/Microsoft.VisualStudio.BlazorExtension/Microsoft.VisualStudio.BlazorExtension.dll",
+        "bin/Microsoft.VisualStudio.LanguageServices.Blazor/net461/Microsoft.VisualStudio.LanguageServices.Blazor.dll",
+      ]
+    },
+    {
+      "certificate": "3PartySHA2",
+      "values": [
+        "bin/Microsoft.AspNetCore.Blazor.Build/netcoreapp2.1/AngleSharp.dll",
+        "bin/Microsoft.VisualStudio.BlazorExtension/AngleSharp.dll"
+      ]
+    },
+    {
+      "certificate": "VsixSHA2",
+      "values": [
+        "bin/Microsoft.VisualStudio.BlazorExtension/Microsoft.VisualStudio.BlazorExtension.vsix"
+      ]
+    },
+    {
+      "certificate": "NuGet",
+      "strongName": null,
+      "values": [
+        "packages/*.nupkg"
+      ]
+    }
+  ],
+  "exclude": [
```

Comment:
This just seems super fraught with peril. Do we have to manually update this file when we add an assembly or add a dependeny? This sounds like a nightmare.

Reply:
Yes and yes. The Arcade team is working on a way to automatically generate these, but that feature isn't ready yet. So for now, you will need to manually specify how to handle new dependencies.

Reply:
what's the ETA on this being reasonable?

Reply:
https://github.com/dotnet/arcade/issues/226 and https://github.com/dotnet/arcade/issues/396 are tracking work to manage this data better. These are currently in the August milestone which wraps up 8/23.

Reply:
Out of interest, how did you obtain the list of assemblies here?

Reply:
SignTool produced an error about unrecognized binaries in packages. On local builds, signtool runs a test and skips the signing part, so if new packages appear or new binaries in packages appear as you make changes, local runs of `build.cmd` will fail.